### PR TITLE
Added format_cut_output function for binning result formatting.

### PR DESCRIPTION
**Pull request by hidekoji**
# Description
Describe your fix here

# Checklist
Make sure you have performed the following items before submitting this pull request.
If not, please describe the reason.  

- [x] Add test cases for this fix/enhancement
- [ ] Pass devtools::check()
- [ ] Pass devtools::test()
- [ ] Test installing from github
- [x] Tested with Exploratory

### CODE REVIEW

**Review by kei51e — APPROVED** [review]
Looks good to me.

File: R/util.R [hunk 1]
```diff
@@ -2920,3 +2920,146 @@ separate_japanese_address <- function(df, address, prefecture_col = "prefecture"
...
+# length as the base.labels.
+#
+#  new.labels <- c("a", "b", "c")
+#  base.labels <- c("1", "2", "3", "4", "5")
+#  _tam_construct_new_labels (base.labels, new.labels)
+#  [1] "a" "b" "c" "4" "5"
+#
+#  new.labels <- c("a", "b", "c", "d", "e", "f")
+#  base.labels <- c("1", "2", "3", "4", "5")
+#  _tam_construct_new_labels (base.labels, new.labels)
```
Comment (hideaki) [inline R/util.R:2936]: Function name in the comment here should be updated.

File: R/util.R [hunk 2]
```diff
@@ -2920,3 +2920,146 @@ separate_japanese_address <- function(df, address, prefecture_col = "prefecture"
...
+#  [1] "a" "b" "c" "d" "e"
+construct_new_labels <- function(base.labels, new.labels) {
+  if (is.null(new.labels)) {
+    base.labels
+  }
+  # If the length of new.labels and base.labels are the same,
+  # new.labels is ready to use. Just return it.
+  else if (length(new.labels) == length(base.labels)) {
+    coalesce(new.labels, base.labels)
```
Comment (hideaki) [inline R/util.R:2945]: Add the namespace to coalesce call.

File: R/util.R [hunk 3]
```diff
@@ -2920,3 +2920,146 @@ separate_japanese_address <- function(df, address, prefecture_col = "prefecture"
...
+  dplyr::recode(x, !!!new.names)
+}
+
+# It formats a given character vector generated by cut function.
+#
+# Example:
+# Input: [1] "(0.996,2.333333333]"       "(2.333333333,3.666666667]" "(3.666666667,5.004]"
+# Output: [1] "1.00 - 2.33" "2.33 - 3.67" "3.67 - 5.00"
+#
+format_cut_output_line <- function(x, decimal.digits=2, big.mark=",", small.mark=".", prefix="", suffix="", right=TRUE) {
```
Comment (hideaki) [inline R/util.R:2994]: Rename it to something like format_cut_output_levels to express what it is better.

File: R/util.R (hunk 2 above)
Comment (hidekoji) [reply R/util.R]: Added "dplyr" namespace.

File: R/util.R (hunk 1 above)
Comment (hidekoji) [reply R/util.R]: Updated function name in the comment.

File: R/util.R [hunk 4]
```diff
@@ -2920,3 +2920,146 @@ separate_japanese_address <- function(df, address, prefecture_col = "prefecture"
...
 }
+
+# Constructs a new vector string for the levels.
+# It generates a new character vector based on the base.labels
+# and overrides it with new.labels. The return value has the same
+# length as the base.labels.
+#
+#  new.labels <- c("a", "b", "c")
+#  base.labels <- c("1", "2", "3", "4", "5")
+#  _tam_construct_new_labels (base.labels, new.labels)
```
[the remark this answers is not in the document]
Comment (hidekoji) [reply R/util.R]: Updated function name in the comment.

File: R/util.R (hunk 3 above)
Comment (hidekoji) [reply R/util.R]: Renamed it to format_cut_output_levels

**Pull request merged**